### PR TITLE
fix(a11y): provide proper aria label for mobile navigation toggle

### PR DESCRIPTION
**Pull request by timbru31**

### CODE REVIEW

File: src/_data/global.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
 	"navigation": {
-		"ariaLabel": "Main",
+		"ariaLabel": "Main navigation",
+		"ariaLabelMobileToggle": "Main navigation mobile toggle",
```
Comment (terabaud) [inline src/_data/global.json:4]: Is it relevant for the toggle button to mention it is a `mobile` toggle or would it be sufficient to label it `Main navigation` or maybe `Toggle main navigation`?

File: src/_includes/navigation.liquid [hunk 2]
```diff
@@ -25,7 +25,7 @@
 				</li>
 				{% endfor %}
 			</ul>
-			<button class="nav__mobile-toggle">
+			<button class="nav__mobile-toggle" aria-label="{{ data.navigation.ariaLabelMobileToggle }}">
```
Comment (terabaud) [inline src/_includes/navigation.liquid:28]: * describe via aria that the button opens a menu which overlaps the content (or parts of it) via `aria-haspopup="true"`
* use `aria-controls="idofthelayer"` to tell which overlay is controlled by the button
* add aria-expanded="false|true" if the layer is closed|open

File: src/_data/global.json (hunk 1 above)
Comment (timbru31) [reply src/_data/global.json:4]: I included the mobile as the div is visually hidden on mobile. Not sure if all screenreaders honor the display: none flag. If yes, the "mobile" part is probably redundant.

Comment (terabaud) [reply src/_data/global.json:4]: `display: none` is considered in the same way as a `aria-hidden="true"` and will not be exposed to screen readers.

FYI if you would want to provide content that is only visible to screenreaders, common practice is to use an utility class to make it visually hidden:
https://github.com/terabaud/lea-codes/blob/main/src/scss/utils/_visually-hidden.scss

Comment (timbru31) [reply src/_data/global.json:4]:
```suggestion
		"ariaLabelMobileToggle": "Toggle main navigation",
```

**Review by moritzbehnke — APPROVED** [review]

**Pull request merged**